### ISSUE TITLE
Watcher not noticing changes - No Debug output

### ISSUE THREAD

**Issue by mrforsythexeter** [liximomo/vscode-sftp#869]
At the moment I have the watcher set to

```
  "watcher" : {
      "files": "**/*.css",
      "autotrue"  : true,
      "autoDelete" : false
  }

I have a terminal open, with a grunt watch (watching for SASS changes) This runs and updates the css files

However nothing happens, no upload of the watched files

I have debug on, but no report of the watched files, and nothing mentioned about detecting file changes


 - OS: Win
 - VSCode Version: 1.52.0
 - Extension Version: 1.12.9

**Extension Logs from Startup** - *required*
[12-15 22:17:47] [debug] register command "Cancel All Transfer" from "./commandCancelAllTransfer.ts"
[12-15 22:17:47] [debug] register command "Config" from "./commandConfig.ts"
[12-15 22:17:47] [debug] register command "List Active Folder" from "./commandListActiveFolder.ts"
[12-15 22:17:47] [debug] register command "Open Ssh Connection" from "./commandOpenSshConnection.ts"
[12-15 22:17:47] [debug] register command "Set Profile" from "./commandSetProfile.ts"
[12-15 22:17:47] [debug] register command "Toggle Output Panel" from "./commandToggleOutputPanel.ts"
[12-15 22:17:47] [debug] register command "Upload Changed Files" from "./commandUploadChangedFiles.ts"
[12-15 22:17:47] [debug] register command "Delete Remote" from "./fileCommandDeleteRemote.ts"
[12-15 22:17:47] [debug] register command "Diff" from "./fileCommandDiff.ts"
[12-15 22:17:47] [debug] register command "Diff Active File" from "./fileCommandDiffActiveFile.ts"
[12-15 22:17:47] [debug] register command "Download" from "./fileCommandDownload.ts"
[12-15 22:17:47] [debug] register command "Download Active File" from "./fileCommandDownloadActiveFile.ts"
[12-15 22:17:47] [debug] register command "Download Active Folder" from "./fileCommandDownloadActiveFolder.ts"
[12-15 22:17:47] [debug] register command "Download File" from "./fileCommandDownloadFile.ts"
[12-15 22:17:47] [debug] register command "Download Folder" from "./fileCommandDownloadFolder.ts"
[12-15 22:17:47] [debug] register command "Download Force" from "./fileCommandDownloadForce.ts"
[12-15 22:17:47] [debug] register command "Download Project" from "./fileCommandDownloadProject.ts"
[12-15 22:17:47] [debug] register command "Edit In Local" from "./fileCommandEditInLocal.ts"
[12-15 22:17:47] [debug] register command "List" from "./fileCommandList.ts"
[12-15 22:17:47] [debug] register command "List All" from "./fileCommandListAll.ts"
[12-15 22:17:47] [debug] register command "Reveal In Explorer" from "./fileCommandRevealInExplorer.ts"
[12-15 22:17:47] [debug] register command "Reveal In Remote Explorer" from "./fileCommandRevealInRemoteExplorer.ts"
[12-15 22:17:47] [debug] register command "Sync Both Directions" from "./fileCommandSyncBothDirections.ts"
[12-15 22:17:47] [debug] register command "Sync Local To Remote" from "./fileCommandSyncLocalToRemote.ts"
[12-15 22:17:47] [debug] register command "Sync Remote To Local" from "./fileCommandSyncRemoteToLocal.ts"
[12-15 22:17:47] [debug] register command "Upload" from "./fileCommandUpload.ts"
[12-15 22:17:47] [debug] register command "Upload Active File" from "./fileCommandUploadActiveFile.ts"
[12-15 22:17:47] [debug] register command "Upload Active Folder" from "./fileCommandUploadActiveFolder.ts"
[12-15 22:17:47] [debug] register command "Upload File" from "./fileCommandUploadFile.ts"
[12-15 22:17:47] [debug] register command "Upload Folder" from "./fileCommandUploadFolder.ts"
[12-15 22:17:47] [debug] register command "Upload Force" from "./fileCommandUploadForce.ts"
[12-15 22:17:47] [debug] register command "Upload Project" from "./fileCommandUploadProject.ts"
[12-15 22:17:47] [info] config at c:\Users\XXXXX\Documents\Projects\XXXXXX\XXXXXXXX {"remotePath":"/var/www/XXXXXXXXXXX/","uploadOnSave":true,"downloadOnOpen":true,"ignore":[],"concurrency":4,"protocol":"sftp","connectTimeout":10000,"interactiveAuth":false,"secure":false,"remoteTimeOffsetInHours":0,"name":"XXXXX Server 1","host":"XXX.XXXX.XXXX","port":XXXX,"username":"******","password":"******","watcher":{"files":"**/*.css","autotrue":true,"autoDelete":false}}

**Comment by zhenshiming** [comment]
auto**true**？

**Comment by mrforsythexeter** [comment]
I am ashamed, sorry I did notice this.. what a wally. I should have #closed #869 glad you know what you are doing.
